README.md

```diff
@@ -109,8 +109,9 @@ Options:
                            default mode.
   -c, --custom TEXT        Validate against a custom schema (local filepath or
                            remote schema).
-  -r, --recursive INTEGER  Recursively validate all related stac objects. A
-                           depth of -1 indicates full recursion.
+  -r, --recursive          Recursively validate all related stac objects.
+  -m, --max-depth INTEGER  Maximum depth to traverse when recursing. Ignored
+                           if `recursive == False`.
```

Comment:
It might be good to document how to get full recursion (if possible).  -1?  Omit the arg?

Reply:
Should get full recursion if you omit the argument: https://github.com/stac-utils/stac-validator/pull/190/files#diff-901a8575e00bc5aeaa6ec79a3ee5ec98cf72e603f8d2c18556310aea8d43cfd6R208.